crates/ironrdp-pdu/src/basic_output/fast_path.rs

```diff
@@ -20,7 +20,7 @@ use crate::{decode_cursor, per, PduDecode, PduEncode, PduResult};
 pub struct FastPathHeader {
     pub flags: EncryptionFlags,
     pub data_length: usize,
-    forced_long_length: bool,
+    pub forced_long_length: bool,
```

Comment:
I would rather keep this private, as an implementation detail. I think it exist for the sole purpose of keeping the round-tripping property even when a "weird" packet is decoded, but it’s not intended to be changed otherwise. Can you confirm? @pacmancoder 
Instead, add a `new` constructor.

Reply:
Agreed, it is not intended to be public